tests/unit-core/test-snapshot.c

```diff
@@ -226,7 +226,7 @@ main (void)
       0x4A, 0x52, 0x52, 0x59, 0x18, 0x00, 0x00, 0x00,
       0x00, 0x00, 0x00, 0x00, 0x48, 0x00, 0x00, 0x00,
       0x01, 0x00, 0x00, 0x00, 0x18, 0x00, 0x00, 0x00,
-      0x03, 0x00, 0x01, 0x00, 0x01, 0x00, 0x01, 0x00,
+      0x03, 0x00, 0x01, 0x00, 0x21, 0x00, 0x01, 0x00,
```

Comment:
If the snapshot changed, shouldn't the snapshot version also change?

Reply:
It is done.